cra_buildspec.yaml

```diff
@@ -15,10 +15,6 @@ phases:
       - echo launching yarn install in $SUB_PRJ_NAME...
       - cd $SUB_PRJ_NAME
       - yarn install
-      - | 
-        if ( [ "$SUB_PRJ_NAME" != "packages/pn-landing-webapp" ] ) then 
-          echo Running tests.. && yarn test
```

Comment:
the condition says that if $SUB_PRJ_NAME **isn't** packages/pn-landing-webapp, tests must be run. so adjust this condition

Reply:
done